canonicalwebteam/discourse_docs/parsers.py

```diff
@@ -282,18 +353,18 @@ def get_section(soup, title_text):
     if not heading:
         return None
 
-    # If there's another heading of the same level
-    # get the content before it
     heading_tag = heading.name
-    next_heading = heading.find_next(heading_tag)
 
+    section_html = "".join(map(str, heading.fetchNextSiblings()))
```

Comment:
Rebase change, or do you want to reverse the previous change?

Reply:
I realised actually this change I made in the last PR created a bug, and the original structure needed to remain.

Reply:
Alright :)